_docs/collators.md

```diff
@@ -74,7 +74,7 @@ Navigate to our other documentation to use the PolkadotJS app to rotate keys. Na
 
 ### Step 2: Set your session keys
 
-**Step 2.1: Head over to the extrinsics tab in PolkadotJS**
+**Step 2.1: Head over to the Developer -> Chain state tab in PolkadotJS**
```

Comment:
I came from the onfinality node setup page, and this is the first mention of the chain state tab, so trying to make its location clearer.